diff --git a/src/qml/About/Main.qml b/src/qml/About/Main.qml
index dc4d0fe..08202d4 100644
--- a/src/qml/About/Main.qml
+++ b/src/qml/About/Main.qml
@@ -25,6 +25,8 @@ import Cutefish.Settings 1.0
 import "../"
 
 ItemPage {
+    property var pixmapsPath: String("file://usr/share/pixmaps")
+    
     headerTitle: qsTr("About")
 
     About {
@@ -48,7 +50,7 @@ ItemPage {
                 width: 140
                 height: 72
                 sourceSize: Qt.size(width, height)
-                source: "qrc:/images/logo.svg"
+                source: pixmapsPath + "/fedora-logo-sprite.svg"
             }
 
             Item {
